pages/community/[slug].tsx

```diff
@@ -35,6 +36,24 @@ const postBody = ({ content, post }) => {
 export default function Post({ post, posts, preview }) {
   const router = useRouter();
   const morePosts = posts?.edges;
+  const postBodyRef = useRef<HTMLDivElement>();
+  const a = useSpringValue(0);
```

Comment:
here too

Reply:
made the changes please refer to the commit- [817c2aa](https://github.com/keploy/blog-website/pull/31/commits/817c2aac2993b1939da6a7049aaf723305525bde)